### PR TITLE
[MIG][12.0] partner_delivery_schedule: Migration to v12.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT20121
@ernestotejeda @rafaelbn review please!!

### CODE REVIEW

File: partner_delivery_schedule/models/partner_delivery_schedule.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class DeliverySchedule(models.Model):
+    _name = 'delivery.schedule'
```
Comment (pedrobaeza) [inline partner_delivery_schedule/models/partner_delivery_schedule.py:8]:
```
WARNING openerp_test odoo.models: The model delivery.schedule has no _description
```

**Review by pedrobaeza — APPROVED** [review]

**Comment by sergio-teruel** [comment]
@pedrobaeza Changes done!!

**Review by ernestotejeda — COMMENTED** [review]
Code review

File: partner_delivery_schedule/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    'name': 'Partner Delivery Schedule',
+    'summary': 'Set on partners a schedule for delivery goods',
+    'version': '12.0.1.0.1',
```
Comment (ernestotejeda) [inline partner_delivery_schedule/__manifest__.py:6]:
```suggestion
    'version': '12.0.1.0.0',
```

**Review by ernestotejeda — COMMENTED** [review]
Maybe you can implement this later by having in re.partner a Many2one with resource.calendar instead of a Many2many with delivery.schedule and so we will have one less model to worry about and maybe the allow_delivery_date method could be shorter.
If it is not done in the migration of this module it could be done as an improvement later. What do you think ?

**Pull request merged**